tsconfig.json

```diff
@@ -26,6 +28,16 @@
             "or": "||"
           }
         }
+      },
+      "ember-template-imports": {
```

Comment:
Does this fix something? We don't use .gts in this addon yet so this is a no op afaik